javascript/listview.js

```diff
@@ -1,81 +1,200 @@
-var divDistanceIcon = document.createElement('div');
-divDistanceIcon.setAttribute("class", "distancefrom-icon");
-var divDistanceNumber = document.createElement('div');
-divDistanceNumber.setAttribute("class", "distancefrom-number");
-divDistanceNumber.textContent = '1.2 mi'
-var imgDistanceIcon = document.createElement('img');
-imgDistanceIcon.setAttribute("class", "nearme");
-imgDistanceIcon.setAttribute("src", "images/nearme.png");
-
-titleAndDistanceSelector.appendChild(divDistanceFrom);
-divDistanceFrom.appendChild(divDistanceIcon);
-divDistanceFrom.appendChild(divDistanceNumber);
-divDistanceIcon.appendChild(imgDistanceIcon);
-
-// Info Section - Review Row
-
-
+var getTitleValue = function(title) {
+    var dbTitleRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/title');
+    dbTitleRef.on('value', snap => title.textContent = snap.val());
+};
+
+var getDistanceFromValue = function(selector) {
+    // This function will need to calculate distance
+    // between tour and user location in the future
+    var calculateDistance = '1.2 mi';
+    selector.textContent = calculateDistance;
+};
+
+var getAmountOfStars = function() {
+    // Creates number of stars based on overall rating
+    // Blocked on getting value out
+
+    // var dbRatingRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/rating');
+    // dbRatingRef.on('value', snap => reviewsSelector.textContent = snap.val());
+
+};
+
+var getNumberOfReviewsValue = function(reviews) {
+    // Filled with placeholder values
+    var dbReviewsRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/reviews');
+    dbReviewsRef.on('value', snap => reviews.textContent = snap.val() + ' Reviews');
+};
+
+var getAddressValue = function(address) {
+    var dbAddressRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/address');
+    var dbCityRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/city');
+    var dbStateRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/state');
+
+    dbAddressRef.on('value', function(snapshot) {
+        address.textContent = snapshot.val();
+        address.textContent += '\r\n';
+    });
+    dbCityRef.on('value', function(snapshot) {
+        address.textContent += snapshot.val();
+        address.textContent += ', ';
+    });
+    dbStateRef.on('value', snap => address.textContent += snap.val());
+};
+
+var getLengthAndTime = function(timeAndLength) {
+    // Filled with placeholder values
+    var dbLengthRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/length');
+    var dbTimeRef = firebase.database().ref('routes/-L7Aa3dtKeXFFX8SLOTF/time');
+    
+    dbLengthRef.on('value', function(snapshot) {
+        timeAndLength.textContent = snapshot.val();
+        timeAndLength.textContent += ' mins / ';
+    });
+    dbTimeRef.on('value', function(snapshot) {
+        timeAndLength.textContent += snapshot.val();
+        timeAndLength.textContent += ' mi';
+    });
```

Comment:
I'll get the estimated time for you.  It is the same process as getting distance.